packages/story-editor/src/components/videoTrim/useVideoTrimMode.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useFeature } from 'flagged';
+import { useCallback, useMemo } from '@web-stories-wp/react';
+
+/**
+ * Internal dependencies
+ */
+import { useCanvas, useStory } from '../../app';
+
+function useVideoTrimMode() {
+  const isVideoTrimEnabled = useFeature('enableVideoTrim');
```

Comment:
We could also check to see if user can transcode video. See `isTranscodingEnabled`. 

https://github.com/google/web-stories-wp/blob/77e354b4ff9f927c0cc15b80f88fd29df16b78e3/packages/story-editor/src/app/media/utils/useFFmpeg.js#L419-L429

No point of showing this functionality if you can transcode.

Reply:
Wait, now I don't have the button anymore. Why would transcoding not be available on my system? Is it because I'm serving over HTTP rather than HTTPS (for the cross-origin isolation stuff)?

Reply:
Would need to make sure that all that cases match. So user can upload, cross origin isloation is enable and user has opt-in into transcoding? Are all these cases true?

Reply:
Fixed the issue with `window.crossOriginIsolated`, thanks!